shared/actions/json/wallets.json

```diff
@@ -425,6 +425,10 @@
       "airdropState": "Types.AirdropState"
     },
     "updateAirdropDetails": {},
-    "updatedAirdropDetails": {"details": "Types.AirdropDetails"}
+    "updatedAirdropDetails": {"details": "Types.AirdropDetails"},
+    "setTrustlineExpanded": {
```

Comment:
i think this kind of thing should likely be kept as local state in the modal. it doesn't need to be in the store i think. can leave it for now but in general